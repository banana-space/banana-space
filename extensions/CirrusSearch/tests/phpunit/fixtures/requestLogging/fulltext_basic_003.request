{
	"type": "fulltext",
	"term": "mani page",
	"showSuggestion": true
}
