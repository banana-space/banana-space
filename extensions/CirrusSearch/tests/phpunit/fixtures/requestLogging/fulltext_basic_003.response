[
    {
        "responses": [
            {
                "took": 26,
                "timed_out": false,
                "_shards": {
                    "total": 8,
                    "successful": 8,
                    "failed": 0
                },
                "hits": {
                    "total": 0,
                    "max_score": null,
                    "hits": []
                },
                "suggest": {
                    "suggest": [
                        {
                            "text": "mani page",
                            "offset": 0,
                            "length": 9,
                            "options": [
                                {
                                    "text": "main page",
                                    "highlighted": "<em>main<\/em> page",
                                    "score": 0.33238623
                                }
                            ]
                        }
                    ]
                }
            }
        ]
    }
]
