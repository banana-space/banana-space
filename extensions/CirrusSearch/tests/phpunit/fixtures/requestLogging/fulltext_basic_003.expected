[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits_offset": 0,
                    "hits_returned": 0,
                    "hits_total": 0,
                    "indices": [
                        "wiki"
                    ],
                    "limit": 21,
                    "query": "mani page",
                    "query_type": "full_text",
                    "search_time_ms": 26,
                    "suggestion": "main page",
                    "suggestion_requested": true,
                    "syntax": [
                        "full_text",
                        "full_text_querystring",
                        "simple_bag_of_words"
                    ]
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 0,
            "hitsTotal": 0,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki",
            "limit": 21,
            "query": "mani page",
            "queryType": "full_text",
            "source": "cli",
            "suggestion": "main page",
            "suggestionRequested": true,
            "syntax": [
                "full_text",
                "full_text_querystring",
                "simple_bag_of_words"
            ],
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset} and suggested '{suggestion}'. Requested via {source} for {identity} by executor {executor}"
    }
]